mage_ai/frontend/playwright.config.ci.ts

```diff
@@ -44,6 +44,7 @@ export default defineConfig({
     cwd: '../../',
     env: {
       PYTHONPATH: '.',
+      REQUIRE_USER_AUTHENTICATION: '1',
```

Comment:
@edmondwinston So the issue with the login credentials failing was that we needed to enable user authentication.